pkg/storage/storage_fs.go

```diff
@@ -1264,6 +1275,90 @@ func (is *ImageStoreFS) DeleteBlob(repo string, digest string) error {
 	return nil
 }
 
+func (is *ImageStoreFS) GetReferrers(repo, digest string, mediaType string) ([]notation.Descriptor, error) {
+	dir := path.Join(is.rootDir, repo)
+	if !is.DirExists(dir) {
```

Comment:
Although I *think* the callers look safe wrt to this right now, I do think it would be worthwhile to add a check to make sure that the filepath.Abs(dir) is a sub-path of is.rootDir.

Reply:
https://github.com/anuvu/zot/issues/300
sounds like a general concern, will do a project-wide review.